src/Containers/RideProgressCard/RideProgressCard.story.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import { storiesOf } from "@storybook/react-native";
+import RideProgressCard from "./RideProgressCard";
+import { View } from "react-native";
+import { object } from "@storybook/addon-knobs/react";
+
+const stories = storiesOf("Ride Progress Card", module);
+
+const defaultProps = [
```

Comment:
defaultProps is not a good name for it, because it has a naming collision with static property of React.Component and can be understood in different way